lib/lob/resources/card.rb

```diff
@@ -0,0 +1,46 @@
+require "lob/resources/resource_base"
+
+module Lob
+  module Resources
+    class Card < Lob::Resources::ResourceBase
+
+      def initialize(config)
+        super(config)
+        @endpoint = "cards"
+      end
+      
+      def update(resource_id, body={}, headers={})
```

Comment:
I noticed that update isn't available for the other resources (which is consistent with the Node SDK as well) so I only added it here. I'm not sure what the reasoning for leaving out update functionality is. However, we added `/cards` update for Node, PHP and Python SDK, so we might as well add it here and be consistent.

Reply:
As long as the endpoint is publically exposable, it should be added.

Reply:
For cards, update is publicly available. And I saw after creating this PR that there are endpoints where update is not publicly available, so I answered my own question (who would have thunk that reading docs would answer one's API questions 🙃 )